### PR TITLE
identities, accounts, and invites for IAM Overview section

### PR DESCRIPTION
**Pull request by CamrynCarter**
Tenancy and account information for IAM Concepts section of the Overview. Contains instructions for inviting a user to your tenancy.

### CODE REVIEW

File: content/docs/overview/identity-and-access-management/index.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+menu: 
+  docs:
+    parent: "overview"
+weight: 5
+toc: true
+---
+
+## Tenancies and Accounts
+
+Each RKVST tenancy represents an organization, and each RKVST account represents an individual user. There may be multiple accounts within a tenancy if there are several members within an organization.
```
Comment (kjc-jitsuin) [inline content/docs/overview/identity-and-access-management/index.md:18]: Can we note that a user can be part of multiple tenancies? 

"Additionally, an individual user can be part of multiple tenancies."

File: content/docs/overview/identity-and-access-management/index.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    "https://app.rkvst.io/archivist/iam/v1/invites"
+```
+
+See instructions for [creating your `BEARER_TOKEN_FILE`](https://docs.rkvst.com/docs/rkvst-basics/getting-access-tokens-using-app-registrations/) here.
+{{< /tab >}}
+{{< /tabs >}}
+
+### Can I name my tenancy? 
+
+There are two ways to name your RKVST tenancy. The first way is to add a `Tenant Display Name`. This name will be displayed only within your own tenancy. The display name makes it easy to identify which tencancy you're currently working in and to switch between tenancies if you are part of multiple.
```
Comment (kjc-jitsuin) [inline content/docs/overview/identity-and-access-management/index.md]: We should show where this is set. It isn't immediately obvious how to get to the text box to edit.

File: content/docs/overview/identity-and-access-management/index.md (hunk 1 above)
Comment (CamrynCarter) [reply content/docs/overview/identity-and-access-management/index.md:18]: Added

File: content/docs/overview/identity-and-access-management/index.md (hunk 2 above)
Comment (CamrynCarter) [reply content/docs/overview/identity-and-access-management/index.md]: Added!

**Comment by kjc-jitsuin** [comment]
Looks good to me!

**Review by kjc-jitsuin — APPROVED** [review]

**Pull request merged**